conan/tools/cmake/cmakedeps/cmakedeps.py

```diff
@@ -101,6 +101,7 @@ def content(self):
         if direct_configs:
             # Some helpful verbose messages about generated files
             msg = ["CMakeDeps necessary find_package() and targets for your CMakeLists.txt"]
+            ret["_conan_direct_list.cmake"] = "set (_conan_declared_names " + ' '.join(c.file_name for c in direct_configs) + ")"
```

Comment:
Only direct ones coming from above for now, and the name is not right for find_mode both afaik